### PR TITLE
DOCS-2320: Webhook testing documentation update.

### PR DESCRIPTION
**Pull request by bartolini**
Product Version(s):
Calico Enterprise 3.20

Issue:
https://tigera.atlassian.net/browse/DOCS-2320
https://tigera.atlassian.net/browse/RS-2090

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-enterprise/threat/configuring-webhooks.mdx [hunk 1]
```diff
@@ -26,3 +26,10 @@ See [Sending messages using Incoming Webhooks](https://api.slack.com/messaging/w
 2. Enter a **Name** for your webhook, select which **Event types** you want to get alerts for, and, under **Type**, select whether to configure the webhook for Slack, Jira, or for generic JSON output.
 3. Complete the fields for your webhook type and click **Create Webhook**.
 
+## Testing a webhook
```
Comment (ctauchen) [inline calico-enterprise/threat/configuring-webhooks.mdx]: Let's move the verification step into the main procedure. Perhaps something like this:

4. To check that your webhook is working, find the webhook on the **Webhooks** page and click **Actions** > **Test Webhook**. If your webhook is set up correctly, you should generate a HTTP response. 
 
image

**Review by ctauchen — COMMENTED** [review]
Suggestion for bringing the step into the procedure.

File: calico-enterprise/threat/configuring-webhooks.mdx (hunk 1 above)
Comment (bartolini) [reply calico-enterprise/threat/configuring-webhooks.mdx]: Done.

**Comment by ctauchen** [comment]
LGTM

**Pull request merged**